recipe/build.sh

```diff
@@ -76,7 +76,7 @@ if [[ "${HOST}" == "${BUILD}" ]]; then
     exit 1
   fi
 fi
-make install_sw
+make install_sw install_ssldirs
```

Comment:
To prevent this happening again, would it be better to use `make install` here? Or is there a reason for not wanting to include the documentation?
```suggestion
make install
```

Reply:
#28 switched to `make install_sw` specifically because the man pages were not desired.  Also see #27.